src/main/java/spoon/reflect/visitor/DefaultJavaPrettyPrinter.java

```diff
@@ -182,9 +185,14 @@
 	private Environment env;
 
 	/**
-	 * Printer helper.
+	 * Token detector, which delegates tokens to {@link PrinterTokenWriter}
+	 */
+	private PrinterTokenDetector printer;
+
+	/**
+	 * The listener which is called before/after each printed node
 	 */
-	private PrinterHelper printer;
+	private PrinterListener printerListener;
```

Comment:
why do you need two helpers, a PrinterTokenDetector and a PrinterListener?

Reply:
PrinterListener is there by mistake. I will remove it